common-pg/src/main/scala/com/socrata/pg/soql/SqlFunctionsLocation.scala

```diff
@@ -24,6 +24,15 @@ trait SqlFunctionsLocation {
     LocationToLatitude -> locationLatLng("latitude"),
     LocationToLongitude -> locationLatLng("longitude"),
     LocationToAddress -> locationAddress _,
+    // TODO: human_address will be null if any address field is null.  But we cannot do a simple coalesce
+    // TODO: without handling the edge case to keep human_address as null when all address fields are indeed null.
+    // The line break in the verbatium string is significant which keep the two sub-fields as
+    // two separate fields like (f1), (f2) instead of (f1, f2)
+    // TODO: May be we should not wrap expression in () to begin with.
+    Location -> formatCall(
+      "ST_AsBinary(%s)" +
+      SqlFragments.Separator +
+      """'{"address": "' || %s ||  '", "city": "' || %s || '", "state": "' || %s || '", "zip": "' || %s || '"}'""".stripMargin) _,
```

Comment:
Hm actually it's construction a SQL string literal containing JSON?  Ick.  Is there any way to do this with a prepared statement placeholder instead?  What happens if one of the "%s"es contains a single-quote?